### ISSUE TITLE
Use case : mixed mode authentication with Identity 3

### ISSUE THREAD

**Issue by maitredede** [OneBitSoftware/Microsoft.AspNetCore.Authentication.ActiveDirectory#5]
Hi,

I would like to create a web app that uses Identity 3 for accounts, with login/password auth, NTLM auth, other middleware auth...
Is it possible to use the NTLM auth like any other middle auth middleware ?

Thanks.

**Comment by SharePointRadi** [comment]
Yes, that is the intent, however there might be some tweaking that you need to do.

**Comment by PeakiesDioda** [comment]
Hello MaitreDede, was you successfull?

**Comment by maitredede** [comment]
Hi @PeakiesDioda ,

No, I was not successfull. The project was a POC, and my company has abandonned it before I can really do something. They have abandonned C# for Java.
